include/bluetooth/gatt.h

```diff
@@ -925,9 +943,12 @@ ssize_t bt_gatt_attr_read_cpf(struct bt_conn *conn,
  *  Helper macro to declare a descriptor attribute.
  *
  *  @param _uuid Descriptor attribute uuid.
- *  @param _perm Descriptor attribute access permissions.
- *  @param _read Descriptor attribute read callback.
- *  @param _write Descriptor attribute write callback.
+ *  @param _perm Descriptor attribute access permissions,
+ *		 a bitmap of enum bt_gatt_perm values.
+ *  @param _read Descriptor attribute read callback
+ *               (bt_gatt_attr_read_func_t).
+ *  @param _write Descriptor attribute write callback
+ *                (bt_gatt_attr_write_func_t).
```

Comment:
Is it possible to refer to a type using doxygen? Perhaps just `@ref bt_gatt_attr_read_func_t`, or something?

Reply:
```suggestion
 *               (@ref bt_gatt_attr_read_func_t).
 *  @param _write Descriptor attribute write callback
 *                (@ref bt_gatt_attr_write_func_t).
```